### ISSUE TITLE
Consider cache-control: immutable

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/tr-design#101]
See:
http://bitsup.blogspot.com.au/2016/05/cache-control-immutable.html

base.css, fixup.js, and the w3c logo are good candidates. This would benefit ReSpec users, as we reload a lot after we make changes.

**Comment by marcoscaceres** [comment]
(fixed fixup.js above)

**Comment by tripu** [comment]
@marcoscaceres, I think no stable version of any of the main browsers supports this yet; is that correct?

I say let's hold on this until there are at least a couple major browsers understanding this extension.

**Comment by marcoscaceres** [comment]
Right, this is for a future release. BTW, what's the release schedule for tr-design?

**Comment by tripu** [comment]
@marcoscaceres, officially https://github.com/w3c/tr-design/blob/gh-pages/README.md#release-process

But we have broken many of the rules we set in the beginning, from deadlines and periods of time, to our use of branches in this repo.

**Comment by marcoscaceres** [comment]
ah, I knew that was somewhere! 

> But we have broken many of the rules we set in the beginning, from deadlines and periods of time, to our use of branches in this repo.

Wait! Is that like a specification at a standards organization where the implementors don't follow the spec?! That's unheard of! 😜

**Comment by tripu** [comment]
@marcoscaceres: do you happen to know what's the status of this extension in browsers now&hellip;?

**Comment by dontcallmedom** [comment]
see "browser support" in https://www.keycdn.com/blog/cache-control-immutable/

>     Firefox – supported since Firefox 49
>     Microsoft Edge – supported since Edge 15
>     Chrome – Not supported as Chrome already has a feature implemented that prevents subresource revalidation on reload
>     Safari – Supported in Safari Technology Preview 24

(data from May)

**Comment by marcoscaceres** [comment]
That seems like pretty good coverage. Better than I expected.

**Comment by tripu** [comment]
Thank you.
I'll raise this proposal with the team; there might be other projects/resources that could benefit from this.

**Comment by tripu** [comment]
I'll experiment locally first a bit with the new extension, and come back with a suggestion for common resources under /TR.

**Comment by marcoscaceres** [comment]
Supported in Chrome since 54:
 https://bugs.chromium.org/p/chromium/issues/detail?id=611416

**Comment by plehegar** [comment]
@deniak , any reason we can't do this?

**Comment by deniak** [comment]
I don't see why not. I'll bring the topic to the systeam and see how we can implement this.

**Comment by deniak** [comment]
We started adding that immutable header to some of the scripts under `/scripts/`:
```shell
HEAD https://www.w3.org/scripts/angularjs/1.3.16/ | grep Cache
Cache-Control: public, max-age=31536000, immutable
```

However, for `fixup.js`, it's a bit tricky as we sometimes update the script under 2016. Same thing for the specifications under /TR. We do allow edit-in-place in some situations.

---

[Comment edited just to fix a couple typos &mdash; @tripu, 8 Oct 2018]
